src/NServiceBus.Testing/Invocations.cs

```diff
@@ -199,6 +199,11 @@ public ExpectedNotDeferMessageInvocation()
 
     class ExpectedReplyToOriginatorInvocation<M> : ExpectedInvocation<ReplyToOriginatorInvocation<M>>
     {
+        public ExpectedReplyToOriginatorInvocation(bool negate = false)
```

Comment:
yep I noticed that too, that code was written that way because other than the typical implementations on the expectations, this one provides it's own check as lambda to access the Sagas properties. Having a dedicated class would probably require some weird workarounds.